### DISCUSSION TITLE
Voice cloning worked well, but can't understand what it is saying

### DISCUSSION THREAD

**Discussion by Myst1ck** [coqui-ai/TTS#2759] [General Q&A]
Hello there everyone!

I have recently trained a VITS model on my own custom dataset, and while the voice cloning worked amazingly well, and it sounds pretty identical, it has trouble speaking the words.
For example, if I give him the default test sentences and monitor them on the tensorboard, I will not be able to understand what it is saying, and it sounds more like gibberish (but in the cloned voice) than the actual sentence.

It was training for around ~8k steps, and while I know this is not a lot, I want to make sure that it is okay that the voice cloning worked, but the voice can't speak well.

If I need to give it more data / fix current data or let it train for more steps it is no problem, but if there is a problem, and that it shouldn't be like that, I will be happy if someone could tell it to me instead of letting it train for more and not fixing the problem.

Thanks for the help!

**Comment by friyin** [comment]
Hi!

I have exactly the same problem voice is cloned OK but I it seems like is speaking a strange language, what can be the problem? Tokenizer issues?

Regards and thanks for help.

**Reply by ysujiang** [reply]
The problem is the same as yours. Has it been resolved？
